app/realtime/card_realtime.rb

```diff
@@ -1,5 +1,5 @@
 class CardRealtime
-  def self.card_moved(card, move_params)
+  def self.card_moved(card, originated_by, move_params)
```

Comment:
Unused method argument - `originated_by`. If it's necessary, use `_` or `_originated_by` as an argument name to indicate that it won't be used.